spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -293,6 +293,9 @@ object SpdxDeclaredLicenseMapping {
         "GNU General Public License (GPL) v. 3" to GPL_3_0_ONLY.toExpression(),
         "GNU General Public License, Version 2 with the Classpath Exception"
                 to (GPL_2_0_ONLY with CLASSPATH_EXCEPTION_2_0),
+        "GNU General Public License, version 2 with the GNU Classpath Exception" to GPL_2_0_OR_LATER.with(
+            CLASSPATH_EXCEPTION_2_0
```

Comment:
Please use `(GPL_2_0_OR_LATER with CLASSPATH_EXCEPTION_2_0)` similar to one line above.

Reply:
Also, I find the commit message misleading as it sound as if you're mapping `GPL-2.0-or-later` to something.

Reply:
Any suggestions?

Reply:
done x2.